There is a warning from golint on template command's test

The template command's unit test is marked by golint during build:
```
==> Running static validations <==

==> Running linters <==
cmd/helm/template_test.go:163:12:warning: if block ends with a return statement, so drop this else and outdent its block (golint)
```

--- a/cmd/helm/template_test.go
+++ b/cmd/helm/template_test.go
@@ -160,9 +160,8 @@ func TestTemplateCmd(t *testing.T) {
 					// had the error we were looking for, this test case is
 					// done
 					return
-				} else {
-					t.Fatalf("expected err: %q, got: %q", tt.expectError, err)
 				}
+				t.Fatalf("expected err: %q, got: %q", tt.expectError, err)
 			} else if err != nil {
 				t.Errorf("expected no error, got %v", err)
 			}

